projects/storefrontlib/feature-flags.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+
+export const FEATURE_FLAG_QUOTES = new InjectionToken<boolean>(
```

Comment:
It should rather be named FEATURE_FLAG_COMMENTS_PRODUCT_SUPPORT.
And in order to have these flags also available in core I suggest to put it into a new folder projects/core/src/features